yarn-project/acir-simulator/src/client/client_execution_context.ts

```diff
@@ -27,23 +26,28 @@ export class ClientTxExecutionContext {
 
   /**
    * Gets the notes for a contract address and storage slot.
-   * Returns note load oracle inputs, which includes the paths and the roots.
+   * Returns note preimages and membershipWitnesses.
```

Comment:
Does this return the witnesses anymore? Its the indices right?
Also may be worth adding a note that the indicies are not passed to the app circuit but rather proxied to the kernel